### PR TITLE
Minor corrections to exercise instructions

### PR DESCRIPTION
**Pull request by jrgnicho**
Some minor changes to update/remove instructions

### CODE REVIEW

File: gh_pages/_source/session5/Simple-PCL-Interface-for-Python.rst [hunk 1]
```diff
@@ -43,15 +43,19 @@ We will create a new catkin workspace, since this exercise does not overlap with
...
+Most of the infrastructure for a ros node has already been completed for you; the focus of this exercise is the perception algorithms/pipeline. The `CMakelists.txt` and `package.xml` are complete and a source file has been provided. At this time we will explore the source code that has been provided - browse the provided `py_perception_node.cpp` file. This tutorial has been modified from training `Exercise 5.1 Building a Perception Pipeline <http://ros-industrial.github.io/industrial_training/_source/session5/Building-a-Perception-Pipeline.html>`__ and as such the C++ code has already been set up.  If something does not make sense, revisit that exercise.  Open up the perception_node.cpp file and look over the filtering functions.
```
Comment (marip8) [inline gh_pages/_source/session5/Simple-PCL-Interface-for-Python.rst]: Change `ros` to `ROS`

File: gh_pages/_source/session5/Simple-PCL-Interface-for-Python.rst [hunk 2]
```diff
@@ -63,22 +67,24 @@ Now that we have converted several filters to C++ functions, we are ready to cal
...
    .. code-block:: bash
 
-            ls
+            ls 
 
 We will not be including ‘perception_msgs’ as a dependency as we will not be creating custom messages in this course. If you wish for a more in depth explanation including how to implement customer messages, here is a good `MIT resource <http://duckietown.mit.edu/media/pdfs/1rpRisFoCYUm0XT78j-nAYidlh-cDtLCdEbIaBCnx9ew.pdf>`__ on the steps taken.
 
 
-#. Open *CMakeLists.txt*. You can open the file in Pycharm or Qt (or you can use nano, emacs, vim, or sublime). Uncomment line 23, and save.
+#. Open *CMakeLists.txt*. You can open the file in OT Creator (preferred) or PyCharm (or you can use nano, emacs, vim, or sublime). Uncomment line 19 or wherever you find "# catkin_python_setup() " and save.
```
Comment (marip8) [inline gh_pages/_source/session5/Simple-PCL-Interface-for-Python.rst]: Let's change the verbiage to something like "open the file in a text editor or IDE. Uncomment the line '# catkin_python_setup()` and save. I don't think there's any need to plug specific IDEs

File: gh_pages/_source/session5/Simple-PCL-Interface-for-Python.rst [hunk 3]
```diff
@@ -63,22 +67,24 @@ Now that we have converted several filters to C++ functions, we are ready to cal
...
 
    .. code-block:: bash
 
             catkin_python_setup()
+            
+   Note: PyCharm can be installed from the **Ubuntu Software** application.
```
Comment (marip8) [inline gh_pages/_source/session5/Simple-PCL-Interface-for-Python.rst]: Remove this line for same reason as above.

**Review by marip8 — CHANGES_REQUESTED** [review]

**Review by Levi-Armstrong — APPROVED** [review]

**Comment by Levi-Armstrong** [comment]
CI is failing though

**Review by marip8 — APPROVED** [review]

**Pull request merged**